docker/maistra-builder_2.3.Dockerfile

```diff
@@ -98,7 +98,7 @@ RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@${GOLANG_PROTOBUF_VE
     go install istio.io/tools/cmd/cue-gen@${ISTIO_TOOLS_SHA} && \
     go install istio.io/tools/cmd/envvarlinter@${ISTIO_TOOLS_SHA} && \
     go install istio.io/tools/cmd/testlinter@${ISTIO_TOOLS_SHA} && \
-    go install istio.io/tools/cmd/protoc-gen-golang-deepcopy@${ISTIO_TOOLS_SHA} && \
+    go install istio.io/tools/cmd/protoc-gen-deepcopy@${ISTIO_TOOLS_SHA} && \
```

Comment:
Okay, I understand.

> My suggestion was to change how this is invoked in the api repo.

It's not possible, because `protoc-gen-deepcopy` is not called explicitly, but `make gen` failes, because of missing `protoc-gen-deepcopy`. It's strange, but I didn't investigate how it works under the hood.